validator/validation/graphics.py

```diff
@@ -124,7 +124,8 @@ def generate_boxplot(validation_run, outfolder, variable, label, values, unit_re
     return [png_filename, svg_filename]
 
 
-def generate_overview_map(validation_run, outfolder, metric, label, values, dc1, dc2, pair_name, unit_ref, lons, lats):
+def generate_overview_map(validation_run, outfolder, metric, label, values, dc1, dc2,
+                          pair_name, unit_ref, lons, lats, draw_grid=True):
```

Comment:
What's with the draw_grid parameter? Do I remember correctly that you mentioned that drawing the grid can be problematic for the Hawaiian testcase?

Reply:
that was for the new plotting image, here I added the switch just because of past experience I thought it could cause problems. Drawing the grid is currently switched off in the tests, but I can try turning it on again.

Reply:
If the grid doesn't make problems in testing, I'd try to keep the tests as close to the ops reality as possible (i.e. with grid if the graphs in ops are with grid). If it breaks the tests, I'd leave the switch in.

Reply:
ok, guess that was all part of finding out what caused the segfaults